recipes/bermuda/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "bermuda" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/bermuda-{{ version }}.tar.gz
+  sha256: d912c1d486902af47f07cb46d3737420c0245551745219a14272655f66660152
+
+build:
+  skip: true  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build: 
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - rust                                # [build_platform != target_platform]
```

```suggestion
    - {{ compiler('rust') }}
```

Why only on cross-compiled? This is needed for the build.